test/benchmarks/unitBench/list.bench.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-restricted-modules */
+const chalk = require('chalk');
+const { List } = require('../../../lib/utils');
+const { createHistogram } = require('perf_hooks');
+
+const iterations = 100;
+const defaultItemsSize = 100000;
+const makeBigArray = (length = defaultItemsSize) => Array.from({ length }).fill(1);
```

Comment:
You use arrow functions here and functions below.  Can we choose one and be consistent throughout the file?

My preference is always `function`s but it's up to you.

Reply:
sure, made them consistent